test/models/test_albef.py

```diff
@@ -30,11 +29,10 @@ def vision_encoder():
     return ALBEFVisionEncoder(
         image_size=4,
         patch_size=4,
-        num_layers=2,
-        num_heads=1,
-        hidden_dim=3,
+        num_hidden_layers=2,
```

Comment:
Unrelated to this PR: I think the typical way of using `autouse` is to automatically enable every test method to use the fixture without actually requesting it in their arguments. But here you are still requesting `vision_encoder` in your test method (e.g., `albef_model`) which defeats the purpose of `autouse`.

Do you want the seed or the encoder object to be autoused? If the former, you'd want to create a fixture for the seed with `autouse` and another for the encoder without `autouse`.

Reply:
For some reason I have to set seed in the same method I initialize the vision encoder, or else the vision encoder output is not deterministic every every I run the tests.